packages/@react-stately/layout/src/TableLayout.ts

```diff
@@ -305,27 +260,31 @@ export class TableLayout<T> extends ListLayout<T> {
       // Add some margin around the loader to ensure that scrollbars don't flicker in and out.
       let rect = new Rect(40,  Math.max(y, 40), (width || this.virtualizer.visibleRect.width) - 80, children.length === 0 ? this.virtualizer.visibleRect.height - 80 : 60);
       let loader = new LayoutInfo('loader', 'loader', rect);
```

Comment:
just wanted to point out that this `isLoading` block doesn't trigger for RAC tables since RAC TableBody doesn't take in `loadingState` (which is good and expected) but if the user coincidentally passes `loadingState='loading'` then this does trigger and messes up the layout by creating the loader layout info that we only actually use in RSP TableView.

I don't think this is really something to worry about probably but just wanted to call it out since this happens since we access the props set on the collection nodes in several places